change/@microsoft-teams-js-7edb88fb-c70a-4a3c-a006-20d9bae87fd6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Removed `appNotification` capability and collateral",
```

Comment:
It would be good to flag this for @jadahiya-MSFT when he is back. If possible, it would be good to just not mention appNotification in our patch notes at all.